Regularization

LeixuADCP opened the pull request.

henkjongbloed committed:
Removed redundant functions

with Bart

henkjongbloed committed:
Merge branch 'master' into plot_helper_functions

henkjongbloed committed:
Modified plot function

Also starting point for taylor expanded velocities

henkjongbloed committed:
Created two new velocity models

Not yet finished

Commit:
TaylorTidalVelocityModel

henkjongbloed committed:
Added cell-based continuity

henkjongbloed committed:
Updated VelocitySolver

henkjongbloed committed:
Update VelocitySolver.m

Changed cross-validation function (now it is possible to omit data of your own choice)

henkjongbloed committed:
Update VelocitySolver.m

Added consistency matrix and sensitivity of testing error to regularization parameters

henkjongbloed committed:
Finalized VelocitySolver

Remains to implement classes (now it's a bit ugly still...)

henkjongbloed committed:
Last changes until 24-1

Changed reg. matrix names

henkjongbloed committed:
Merge branch 'master' of https://github.com/bartverm/adcptools

henkjongbloed committed:
Merge branch 'master' into Regularization

henkjongbloed committed:
Update demo.m

henkjongbloed committed:
17-2

Small changes to readDeployment and TidalTaylor

henkjongbloed committed:
Created deprecated folder

henkjongbloed committed:
Added solveroptions and modelparameters classes

henkjongbloed committed:
Renamed solver folder

henkjongbloed committed:
Created SolverOptions and ModelParameters

henkjongbloed committed:
partly modified get_parameters

henkjongbloed committed:
Updated Regularization class

To do still:
- Improve subtidal feedback of cross-correlations of tidal constituents (up to first order)

henkjongbloed committed:
Deleted deprecated scripts

henkjongbloed committed:
modified TaylorTidal

henkjongbloed committed:
Updated data_model

Updated several scripts
Included rotation matrices
TaylorTidal is subclass of DataModel and has properties Taylor and Tidal

henkjongbloed committed:
Finished Regularization

To do: Finish RegularizedSolver

henkjongbloed committed:
Update ModelParameters.m

henkjongbloed committed:
Merge branch 'Regularization' into update_get_parameters

henkjongbloed committed:
Finished RegularizedSolver

To do: rotation

henkjongbloed committed:
Merge branch 'update_get_parameters' into Regularization

henkjongbloed committed:
Merge branch 'master' of https://github.com/bartverm/adcptools

henkjongbloed committed:
Merge branch 'master' into Regularization

henkjongbloed committed:
Update LocationBasedRegularizedSolver.m

henkjongbloed committed:
Failed update of readViseaExtern.m

Ask Bart for help

henkjongbloed committed:
Added simple plotting function

Small change to readDeployment to fix bug

henkjongbloed committed:
Fixed bug in inheritance npars

henkjongbloed committed:
Update RegularizedSolver.m

henkjongbloed committed:
Update RegularizedSolver.m

henkjongbloed committed:
Modified WaterLevel

henkjongbloed committed:
Minor details solvers

henkjongbloed committed:
removed solver folder

LeixuADCP committed:
Update Regularization.m

A few changes

bartverm commented:
I think the branch is still not ready for merge, but thanks for committing the latest changes!

henkjongbloed commented:
I agree! Will look at it this afternoon!

Best, Henk
________________________________
Van: Bart Vermeulen ***@***.***>
Verzonden: Wednesday, March 22, 2023 10:08:46 AM
Aan: bartverm/adcptools ***@***.***>
CC: Henk Jongbloed ***@***.***>; Review requested ***@***.***>
Onderwerp: Re: [bartverm/adcptools] Regularization (PR #27)


I think the branch is still not ready for merge, but thanks for committing the latest changes!

—
Reply to this email directly, view it on GitHub<https://github.com/bartverm/adcptools/pull/27#issuecomment-1479158130>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AP2MCVDRRHDS3CVWNDKLXC3W5K6R5ANCNFSM6AAAAAAWDPJ6XE>.
You are receiving this because your review was requested.Message ID: ***@***.***>

henkjongbloed committed:
Update ModelParameters.m

Goal: Add CrossValidation and SensitivityAnalysis

henkjongbloed committed:
Merge branch 'master' of https://github.com/bartverm/adcptools

henkjongbloed committed:
Merge branch 'master' into Regularization

henkjongbloed committed:
Modified VelocitySolver

bartverm committed:
Merge branch 'master' into Regularization

henkjongbloed committed:
Merge branch 'master' of https://github.com/bartverm/adcptools

henkjongbloed committed:
Merge branch 'master' into Regularization

henkjongbloed committed:
Merged RegularizedSolver into Solver

henkjongbloed committed:
Update Solver.m

henkjongbloed committed:
Updated Solver - added output class ModelParameters

bartverm committed:
Merge branch 'master' into Regularization

henkjongbloed committed:
Update Regularization.m

henkjongbloed committed:
Merge branch 'Regularization' of https://github.com/bartverm/adcptools into Regularization

henkjongbloed committed:
Merge branch 'master' of https://github.com/bartverm/adcptools

henkjongbloed committed:
Merge branch 'master' into Regularization

LeixuADCP committed:
Add aspect ratio to adjust quiver plot

To make quiver look better, add aspect ratio for adjusting the vertical expansion.  Also use the FixAspectRatio to determine if it use real axis equal.
Here is an example:
~.plot([u, v, w],'AspectRatio',20,'FixAspectRatio',true)

iniesten committed:
Update Regularization.m

Removed abundant input arguments ("opts") in several functions. @henkjongbloed please check.

iniesten committed:
Update Regularization.m

Fixed minor bug in regularization.m

iniesten committed:
Update RegularizedSolver.m

1. Changed property-name "opts" to "Ropts" -> "opts" is already a property-name within base class Solver.m and therefore not allowed.
2. Changed access to functions "solve", "solve_single" and "reorder_model_matrix" to protected to comply with base class Solver.m

iniesten committed:
Update TaylorTidalModel.m

Fix mistake in TaylorTidalModel. Model matrix is now the result of SxT (taylor model x tidal model) instead of TxS, corresponding to obj.names.

bartverm committed:
Merge branch 'master' into Regularization

henkjongbloed committed:
Moved some functions to deprecated

henkjongbloed committed:
Merge branch 'Regularization' of https://github.com/bartverm/adcptools into Regularization

henkjongbloed committed:
Update ModelParameters.m

Prepare to add cross-validation functions

henkjongbloed committed:
Updated ModelParameters.m

bartverm committed:
Merge branch 'master' into Regularization

bartverm committed:
Merge branch 'master' into Regularization
